src/constants/tokenTypeColumns.js

```diff
@@ -0,0 +1,124 @@
+import { TOKEN_TYPE_ERC_20, TOKEN_TYPE_ERC_721 } from '@/constants/tokenType'
+import { wrappedFormat } from '@/utils'
+
+export const tokenTypesColumns = {
+  [TOKEN_TYPE_ERC_20]: [
```

Comment:
I prefer using different components rather than of dinamically generating the columns, they are harder to maintain, debug and add styles